client/gutenboarding/gutenberg-styles/README.md

```diff
@@ -0,0 +1,3 @@
+# Gutenberg core stylesheets
+
+From [`WordPress/gutenberg/assets/stylesheets`](https://github.com/WordPress/gutenberg/tree/399e5e143a62a66101285de623cf47fab159bbdb/assets/stylesheets) on Gutenberg v6.6.0.
```

Comment:
Should we add a `TODO` to state that we'll replace the copy pasta with `@wordpress/base-styles` once https://github.com/WordPress/gutenberg/pull/17883 is merged and the package has been published?

Reply:
Good idea!  https://github.com/Automattic/wp-calypso/pull/36525/commits/f935c939426f31dde878b01c135104ff3e5c8851